stages/org.osbuild.pam_limits.conf

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3
+"""
+Create pam_limits module configuration.
+
+This stage creates a pam_limits module configuration file with the given name in
+/etc/security/limits.d. Provided list of configuration directives is written
+as separate lines into the configuration file. At least one configuration
+directive must be specified.
+"""
+
+
+import sys
+
+import osbuild.api
+
+
+SCHEMA = r"""
+"definitions": {
+  "configuration": {
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["domain", "type", "item", "value"],
+    "description": "pam_limits module configuration directive representing one line in the configuration.",
+    "properties": {
+      "domain": {
+        "type": "string",
+        "description": "The file system path type."
+      },
+      "type": {
+        "type": "string",
+        "description": "Absolute file system path.",
+        "enum": ["hard", "soft", "-"]
+      },
+      "item": {
+        "type": "string",
+        "description": "The file access mode when creating the file or directory.",
+        "enum": [
+          "core",
+          "data",
+          "fsize",
+          "memlock",
+          "nofile",
+          "rss",
+          "stack",
+          "cpu",
+          "nproc",
+          "as",
+          "maxlogins",
+          "maxsyslogins",
+          "nonewprivs",
+          "priority",
+          "locks",
+          "sigpending",
+          "msgqueue",
+          "nice",
+          "rtprio"
+        ]
+      },
+      "value": {
+        "oneOf": [
+          {
+            "type": "string",
+            "enum": ["unlimited", "infinity"]
+          },
+          {
+            "type": "integer"
+          }
+        ],
+        "description": "The user to use for the file or directory."
```

Comment:
Looks like the property descriptions were copied from tmpfiles.d? :)

Reply:
Oh yeah, thanks and sorry.... I should have checked this more thoroughly 😕

Reply:
hah, no worries.  I think this one in particular happens to all of us constantly.

Reply:
OK, I feel a bit better about it now... 😄 Descriptions are fixed now.